core/controllers/admin.py

```diff
@@ -848,3 +848,16 @@ def post(self):
             self.render_json({})
         else:
             raise self.InvalidInputException('This app cannot send emails.')
+
+
+class InteractionsByExplorationId(base.BaseHandler):
+    """Handler to show a list of interaction id's for a exploration id."""
+
+    @acl_decorators.can_access_admin_page
+    def get(self):
+        exploration_id = self.request.get('exploration_id')
+        exploration = exp_fetchers.get_exploration_by_id(exploration_id)
+        interaction_ids = {
```

Comment:
Are we expecting this to be a list? (If yes then why `{`?)

Reply:
we want de-duplicated items that's why used a set

Reply:
I just realized that this will create a `set`!